Chaincase.Common/Global.cs

```diff
@@ -152,8 +154,11 @@ public async Task InitializeNoWalletAsync(CancellationToken cancellationToken =
                 if (Config.UseTor)
                 {
                     await TorManager.StartAsync(ensureRunning: false, DataDir);
-                    //TorManager.CreateHiddenServiceAsync();
                     Logger.LogInfo($"{nameof(TorManager)} is initialized.");
+                    var cts = new CancellationToken();
+                    P2EPServer.StartAsync(cts);
```

Comment:
because this creates a listener, we should aim to put this in some lifecycle outside of the primary one.

Reply:
maybe ReceiveViewModel